test/aws-xray-sdk/tc_plugin.rb

```diff
@@ -43,6 +44,7 @@ def test_ec2_metadata_v2_successful
         ami_id: 'ami-03cca83dd001d4666'
       }
     }
+    # We should probably use `assert_equal` here ? Always true otherwise...
     assert expected, XRay::Plugins::EC2.aws
```

Comment:
Not related to my PR, but `assert` is always returning true here... regardless what is passed

We probably want to use `assert_equal` if we want to compare something `expected` with `XRay::Plugins::EC2.aws`